joblib/__init__.py

```diff
@@ -130,9 +130,10 @@
 from .parallel import register_parallel_backend
 from .parallel import parallel_backend
 from .parallel import effective_n_jobs
+from .shelf import JoblibShelf, shelve, shelve_mmap
```

Comment:
Should we expose to the user JoblibShelf? It seems to me that it should be internal.

Reply:
Indeed, will change that